Source/hydro/ppm_nd.F90

```diff
@@ -1267,19 +185,26 @@ subroutine ca_ppm_reconstruct_cuda(lo, hi, &
 
                 end if
 
-                qp(i  ,j,k,n,1) = sp
-                qm(i+1,j,k,n,1) = sm
+                if (put_on_edges == 1) then
```

Comment:
Can you explain, for my edification, what this option does?

Reply:
put_on_edges will take the left and right values of the parabola and use them to initialize the left and right interface states.  I'm open to a different name.